src/pages/signin/SignInPageLayout/index.js

```diff
@@ -35,34 +36,48 @@ const SignInPageLayout = (props) => {
         </SignInPageContent>
     );
 
+    const hasRedirect = !_.isEmpty(backgroundStyle.redirectUri);
+
+    const graphicLayout = (
+        <Pressable
+            style={[
+                styles.flex1,
+                StyleUtils.getBackgroundColorStyle(backgroundStyle.backgroundColor),
+            ]}
+            onPress={() => {
+                Link.openExternalLink(backgroundStyle.redirectUri);
+            }}
+            disabled={!hasRedirect}
+        >
+            <SVGImage
+                width="100%"
+                height="100%"
+                src={backgroundStyle.backgroundImageUri}
+                resizeMode={props.isMediumScreenWidth ? 'contain' : 'cover'}
+            />
+        </Pressable>
+    );
+
     if (props.isSmallScreenWidth) {
         return content;
     }
 
-    const hasRedirect = !_.isEmpty(backgroundStyle.redirectUri);
+    if (props.isMediumScreenWidth) {
+        return (
+            <View style={[styles.dFlex, styles.signInPageInner, styles.flexColumnReverse, styles.justifyContentBetween]}>
+                {props.windowHeight >= variables.minHeigthToShowGraphics && graphicLayout}
```

Comment:
I think the `variables.minHeigthToShowGraphics` should be along with the `props.isMediumScreenWidth` because this causes some unnecessary glitch.  Refer video below

### Desktop

https://user-images.githubusercontent.com/85645967/167669702-35139da3-efc7-46d8-a584-28b42786a3f1.mov

### Web on the right side.
https://user-images.githubusercontent.com/85645967/167669102-f6968ac2-9b87-4daf-82a7-db2b3085e289.mov